### ISSUE TITLE
BUG: Filter menu not working

### ISSUE THREAD

**Issue by nancikennedy** [emptyrivers/wow-remix-checklist#5]
1x RemixCheckList/mixins.lua:96: attempt to call global 'EasyMenu' (a nil value)
[string "@RemixCheckList/mixins.lua"]:96: in function <RemixCheckList/mixins.lua:95>

Locals:
self = Button {
}
(*temporary) = nil
(*temporary) = <table> {
 1 = <table> {
 }
 2 = <table> {
 }
 3 = <table> {
 }
 4 = <table> {
 }
 5 = <table> {
 }
 6 = <table> {
 }
 7 = <table> {
 }
 8 = <table> {
 }
}
(*temporary) = RemixChecklistFrameOptionsMenu {
 point = "TOPLEFT"
 Right = RemixChecklistFrameOptionsMenuRight {
 }
 Left = RemixChecklistFrameOptionsMenuLeft {
 }
 relativePoint = "TOPRIGHT"
 Button = RemixChecklistFrameOptionsMenuButton {
 }
 Icon = RemixChecklistFrameOptionsMenuIcon {
 }
 Middle = RemixChecklistFrameOptionsMenuMiddle {
 }
 Text = RemixChecklistFrameOptionsMenuText {
 }
}
(*temporary) = RemixChecklistFrame {
 OptionsMenu = RemixChecklistFrameOptionsMenu {
 }
 layoutType = "ButtonFrameTemplateNoPortrait"
 dataProvider = <table> {
 }
 scrollView = <table> {
 }
 RefreshButton = Button {
 }
 CloseButton = Button {
 }
 OptionsButton = Button {
 }
 TitleContainer = Frame {
 }
 Bg = RemixChecklistFrameBg {
 }
 NineSlice = Frame {
 }
 ScrollBar = Frame {
 }
 ScrollBox = Frame {
 }
}
(*temporary) = 0
(*temporary) = 0
(*temporary) = "MENU"
(*temporary) = "attempt to call global 'EasyMenu' (a nil value)"
ns = <table> {
 cache = Frame {
 }
 invalidate = false
 saved = <table> {
 }
 data = <table> {
 }
 enum = <table> {
 }
 itemIDs = <table> {
 }
}

**Comment by emptyrivers** [comment]
Yeah, EasyMenu was removed with 11.0. Given that Remix is almost over, I don't think I'll update until the next remix season.
